test/test_torch.py

```diff
@@ -5900,6 +5901,54 @@ def test_isinf_type(self, device):
         with self.assertRaises(TypeError):
             torch.isinf(1)  # Parameter must be a tensor
 
+    @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
+    @dtypes(*tuple(itertools.combinations_with_replacement(torch.testing.get_all_dtypes(include_bfloat16=False), 2)))
```

Comment:
excluding bfloat16 because numpy doesn't support it. If you think it's worth explicit testing I can make a separate test for it that doesn't use numpy to check the outputs.

Reply:
You can check it with NumPy by upcasting it to float before calling NumPy, then converting it back to torch and bfloat16. A helper function might make this easy.